lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -412,19 +428,44 @@ def execute
         untar_export
 
         export_json = read_json
-
         cv = content_view(export_json['name'], options['option_organization_id'])
-        sync_repositories(export_json['repositories'], options['option_organization_id'])
-
-        publish(
-          cv['id'],
-          export_json['major'],
-          export_json['minor'],
-          repos_units(export_json['repositories'])
-        )
+
+        if export_json['composite_components']
+          composite_version_ids = export_json['composite_components'].map {|component| find_local_component_id(component) }
+
+          resource_content_views.call(:update,
+                                      'id' => cv['id'], 'component_ids' => composite_version_ids)
+          publish(cv['id'], export_json['major'], export_json['minor'], nil)
```

Comment:
You could update the publish method signature to set `repo_units = nil` as a default and drop the trailing nil here.